externs/shaka/net.js

```diff
@@ -118,6 +118,8 @@ shaka.extern.Request;
  *   The original URI passed to the browser for networking. This is before any
  *   redirects, but after request filters are executed.
  * @property {BufferSource} data
+ *   The response HTTP status code.
+  * @property {number} status
```

Comment:
Should be ` * @property {(number|undefined)} status`  In the offline plugin it can be optional.

Reply:
I think you are missing a part in the definition, you should add `* status: (number | undefined)` in line 107

Reply:
Thank you very much for your comments, I was missing this!